website/src/styles/Theme/components/Card.ts

```diff
@@ -9,11 +9,10 @@ export const cardTheme = defineMultiStyleConfig({
     return {
       container: {
         backgroundColor: isLightMode ? "white" : "gray.700",
+        p: "6",
```

Comment:
Do we really want this behavior? Personally, I feel using the extra `CardBody` is quite verbose. But I'm seeing many other frameworks use it as well

Reply:
maybe to be more responsive we can use the values `[2, 3, 4, 5, 6]` for all break points?

Reply:
@rsandb can you comment on these suggestions?

Reply:
I prefer the response padding values on the `CardBody` component option. If we want padding on the `Card` component we may need to check if there are any usage of `CardBody` to avoid duplicate padding or set the padding to 0 cause Charka already has default padding in the `CardBody` component.

Reply:
@rsandb So let's add the padding Card component and reset the padding of CardHeader, Body, and Footer

Reply:
Will approve after @notmd's request has been implemented.

Reply:
> @rsandb So let's add the padding Card component and reset the padding of CardHeader, Body, and Footer

I went with your first suggestion so Divider elements don't get shrunk inside the card element